charts/trust-policies/values.yaml

```diff
@@ -25,5 +25,10 @@ policy:
   trust:
     # trust the GitHub signing authority
     github: true
+    # githubTrustDomain may be set to the trust domain applicable for some GitHub Enterprise Cloud accounts.
+    # If applicable, this may be set to the output of the following command:
+    # $ gh api meta | jq .domains.artifact_attestations.trust_domain
```

Comment:
```suggestion
    # $ gh api meta -q .domains.artifact_attestations.trust_domain
```

Reply:
(turns out gh ships with jq?)

Reply:
Good call! I pushed a new commit using the long form flag `--jq`.